addons/binding/org.openhab.binding.astro/ESH-INF/thing/channels.xml

```diff
@@ -125,7 +124,7 @@
 				<option value="SUMMER">Summer</option>
 				<option value="AUTUMN">Autumn</option>
 				<option value="WINTER">Winter</option>
-        	</options>
+			</options>
```

Comment:
Hmmm, i've downloaded the channels.xml, opened it in a text editor and enabled 'show invisibles'. Can't find the extra spaces...